apps/coordinator/ts/health/types.ts

```diff
@@ -0,0 +1,66 @@
+import { IGetZkeyFilePathsData } from "../file/types";
+
+/**
+ * Interface for checking the rapidsnark setup
+ */
+export interface ICheckRapidsnark {
+  /**
+   * Rapidsnark executable path
+   */
+  rapidsnarkExecutablePath: string;
+
+  /**
+   * Is rapidsnark executable
+   */
+  rapidsnarkIsExecutable: boolean;
+}
+
+/**
+ * Interface for checking the zkeys directory
+ */
+export interface ICheckZkeysDirectory {
+  /**
+   * Does zkeys directory exist
+   */
+  zkeysDirectoryExists: boolean;
+
+  /**
+   * Available zkeys
+   */
+  availableZkeys: Record<string, IGetZkeyFilePathsData | boolean>;
+}
+
+/**
+ * Interface for checking the coordinator wallet funds
+ */
+export interface ICheckWalletFunds {
+  /**
+   * Coordinator address
+   */
+  address: string;
+
+  /**
+   * Funds in all networks
+   */
+  fundsInNetworks: { network: string; balance: string; status: boolean }[];
```

Comment:
Let's move `{ network: string; balance: string; status: boolean }` to a separate interface